boards/BMS/Src/App/states/App_AllStates.c

```diff
@@ -189,7 +158,9 @@ bool App_AllStatesRunOnTick100Hz(struct StateMachine *const state_machine)
     {
         acc_meas_settle_count++;
     }
-    else if (App_SharedErrorTable_HasAnyCriticalErrorSet(error_table))
+
+    // App_SharedErrorTable_HasAnyCriticalErrorSet left in for now, to be removed entirely in future
+    else if (acc_fault || ts_fault || App_SharedErrorTable_HasAnyCriticalErrorSet(error_table))
```

Comment:
Error table check left in for now, will be removed once the rest of the BMS faults are moved out of the error table in a future PR. See [ELEC-279](https://ubcformulaelectric.atlassian.net/browse/ELEC-279).